src/WCSimLIGen.cc

```diff
@@ -0,0 +1,282 @@
+#include "WCSimLIGen.hh"
+#include "WCSimPrimaryGeneratorMessenger.hh"
+#include "WCSimPrimaryGeneratorAction.hh"
+
+#include "json.hpp"
+
+#include "G4ParticleGun.hh"
+#include "G4PhysicalConstants.hh"
+#include "G4OpticalPhoton.hh"
+#include "G4RandomDirection.hh"
+#include "G4SystemOfUnits.hh"
+
+using namespace std;
```

Comment:
```suggestion
```
No thanks (HK coding convention recommendation. It'll help you remove the `cout`s too)